tools/cldr-code/src/main/java/org/unicode/cldr/util/PathDescription.java

```diff
@@ -510,10 +511,45 @@ public class PathDescription {
                     + "Sample names for person name format examples (enter ∅∅∅ for optional unused fields). For more information, please see "
                     + CLDRURLS.PERSON_NAME_FORMATS
                     + ".\n"
-                    + "^//ldml/numbers/([a-z]*)Formats(\\[@numberSystem=\"([^\"]*)\"])?/\\1FormatLength/\\1Format\\[@type=\"standard\"]/pattern\\[@type=\"standard\"]$"
+                    + "^//ldml/numbers/rationalFormats/rationalPattern"
                     + RegexLookup.SEPARATOR
-                    + "Special pattern used to compose {1} numbers. Note: before translating, be sure to read "
-                    + CLDRURLS.NUMBER_PATTERNS
+                    + "A pattern that is used to format a rational fraction (eg, ½), using the numerator and denominator"
+                    + CLDRURLS.NUMBERS_HELP
+                    + ".\n"
+                    + "^//ldml/numbers/rationalFormats/integerAndRationalPattern\\[@alt=\"([^\"]*)\"]"
+                    + RegexLookup.SEPARATOR
+                    + "A pattern that is used to “glue” an integer and a formatted rational fraction (eg, ½) together; only used when the rational fraction does not start with an un-superscripted digit"
```

Comment:
"does not start with an un-superscripted digit" -- meaning (1) it does start with a superscripted digit, or (2) it starts with a non-digit? If it starts with a non-digit, would it be something like "one/half"? Just curious...

Reply:
Formally https://util.unicode.org/UnicodeJsps/list-unicodeset.jsp?a=%5Cp%7Bgc%3DNd%7D, we'll need to explain it. Once we have markdown in examples, it would be easy to link out to an explanation